Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using AmazonGameLift.Editor;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Resources.EditorWindow
+{
+    public class GameLiftPlugin : UnityEditor.EditorWindow
+    {
+        [SerializeField] private Texture icon;
+
+        private VisualTreeAsset _mVisualTreeAsset;
+        private VisualElement _root;
+        private VisualElement _currentTab;
+        private List<Button> _tabButtons;
+        private List<VisualElement> _tabContent;
+        private readonly TextProvider _textProvider = TextProviderFactory.Create();
+
+        private const string TabContentSelectedClassName = "TabContent--selected";
+        private const string TabButtonSelectedClassName = "TabButton--selected";
+        private const string TabButtonClassName = "TabButton";
+        private const string TabContentClassName = "TabContent";
+
+        private static GameLiftPlugin GetWindow()
+        {
+            var inspectorType = Type.GetType("UnityEditor.GameView,UnityEditor.dll");
+            var window = GetWindow<GameLiftPlugin>(inspectorType);
+            window.titleContent = new GUIContent("Amazon GameLift", window.icon);
+            return window;
+        }
+
+        [MenuItem("Amazon GameLift/Show Amazon GameLift Window", priority = 0)]
+        public static void ShowWindow()
+        {
+            GetWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Bring Panel to Front", priority = 1)]
+        public static void FocusPanel()
+        {
+            ShowWindow();
+        }
+
+        [MenuItem("Amazon GameLift/Set AWS Account Profiles", priority = 100)]
+        public static void OpenAccountProfilesTab()
+        {
+            GetWindow().OpenTab("Credentials");
```

Comment:
Should we be using the strings defined in `Strings.cs`. We shouldn't be hardcoding strings. If in the future this were to change, we'd need to update multiple instances which isn't the most convenient.

Reply:
Created static class with const strings of available page names